CHANGELOG.md

```diff
@@ -1,3 +1,112 @@
+
+[commit_fde4a75]: https://github.com/OP-TEE/optee_os/commit/fde4a75
+
+## New features
+
+* New supported platforms:
+	* Freescale i.MX6 Quad SABRE Lite & SD
+	* HiSilicon D02
+	* Raspberry Pi3
+	* Renesas RCAR H3
+
+* Pager: Support paging of read/write pages by encrypting them with AES-GCM.
+  Support paging of user TAs. Add global setting for TZSRAM size
+  (CFG_CORE_TZSRAM_EMUL_SIZE), defaults to 300K.
+
+* Support for more than 8 CPU cores
+
+* Added SPI framework and PL022 driver
+
+* GPIO: framework supports multiple instances, PL061 driver now has get/set
+  interrupt and mode control functions
+
+* Secure storage: Encrypt the File Encryption Key with a TA-specific key for
+  better TA isolation. Add build-time and run-time support for multiple storage
+  backends. Add SQLite backend.
+
+* Trusted User Interface: some code is introduced to support the implementation
+  of TUI. This includes: a generic framebuffer driver, display and serial
+  abstractions, and drivers for PL111 (LCD) / PL050 (KMI) / TZC400 and PS2
+  mouse.
+
+* AES acceleration using ARMv8-A Cryptographic Extensions instructions is
+  now supported in AArch32 mode
+
+* Add support for GCC flags: -fsanitize=undefined and -fsanitize=kernel-address
+
+* Use a global setting for core heap size (CFG_CORE_HEAP_SIZE), 64K by default.
+
+* Add macros to unwind and print the call stack of TEE core
+
+* Libtomcrypt: sync with the latest `develop` branch.
+
+* The Trusted Application SDK (ta_dev_kit.mk) can produce libraries (.a)
+
+* Rework assertions and TEE core panics and properly honor NDEBUG
+
+## Bug fixes
+
+* Fix incorrect algorithm passed to cipher.final()
+
+* scripts: support Python 2.x and 3.x
+
+* Secure storage: Add proper locking to support concurrent access. Fix sign
+  extension bug with offset parameter of syscall storage_obj_seek which could
+  cause errors in Aarch32 mode. Fix reading beyond end of file.
+
+* Aarch64: mask all maskable exceptions before doing a normal return from call.
+
+* Device Tree: add no-map property to OP-TEE node in reserved-memory.
+
+* LibTomcrypt: fix CVE-2016-6129
+
+## Known issues
+
+* New issues open on GitHub
+  * [#1081][issue1081] Bad mapping of TA secure memref parameters
+  * [#1071][issue1071] __data_end may not correctly represent text start position when using CFG_WITH_PAGER
+  * [#1069][issue1069] armv7/Aarch32: crash in stack unwind (DPRINT_STACK())
+
+## Tested on
+
+In the list below, _standard_ means that the `xtest` program passed with
+its default configuration, while _extended_ means it was run successfully
+with the additional GlobalPlatform™ TEE Initial Configuration Test Suite
+v1.1.0.4.
+
+<!-- ${PLATFORM}-${PLATFORM_FLAVOR}, ordered alphabetically -->
+* d02: extended
+* hikey: extended
+* imx-mx6qsabrelite: TBD
+* imx-mx6qsabresd: TBD
+* imx-mx6ulevk: TBD
+* ls-ls1021atwr: TBD
+* mediatek-mt8173: TBD
+* rcar-h3: TBD
+* rpi3: TBD
```

Comment:
Just tested and RPi3 seems to be working. To help you a bit I've generated a stable manifest file here. Do you want me to submit that as a PR to build.git?

```xml
<?xml version="1.0" encoding="UTF-8"?>
<manifest>
  <remote fetch="https://github.com/mirror" name="busybox_mirror"/>
  <remote fetch="https://github.com/linaro-swg" name="linaro-swg"/>
  <remote fetch="https://github.com/OP-TEE" name="optee"/>
  
  <project name="arm-trusted-firmware.git" path="arm-trusted-firmware" remote="linaro-swg" revision="ec4262bbeba3c7db609fb2aa77a4ddc6deeeab5f" upstream="rpi3_initial_drop"/>
  <project name="build.git" path="build" remote="optee" revision="c14e9526456622614dd4b79cdbf0a583aabd23e0" upstream="master">
    <linkfile dest="build/Makefile" src="rpi3.mk"/>
    <linkfile dest="build/pi3.cfg" src="rpi3/debugger/pi3.cfg"/>
    <linkfile dest="build/gdb" src="../toolchains/aarch64/bin/aarch64-linux-gnu-gdb"/>
  </project>
  <project name="busybox.git" path="busybox" remote="busybox_mirror" revision="dbf5a6da6a4295ce26edd1ce34fde567d19afa02"/>
  <project name="gen_rootfs.git" path="gen_rootfs" remote="linaro-swg" revision="7618525ca98116a75a8524a8a4d5b6cd8f9e2cb6" upstream="master"/>
  <project name="hello_world.git" path="hello_world" remote="linaro-swg" revision="4725dcc0e02c9e4f131e0d65db6617dfc782f55a" upstream="master"/>
  <project name="linux.git" path="linux" remote="linaro-swg" revision="cc225a78910c37d78f8a00c80dcbf59ef7762884" upstream="electron752-rpi3-optee"/>
  <project name="optee_client.git" path="optee_client" remote="optee" revision="658ae538f76a2624b7f9c40539a600d281d872b4" upstream="master"/>
  <project name="optee_os.git" path="optee_os" remote="optee" revision="287359f44187396c3e12d1bab257ce5918a58cb0" upstream="master"/>
  <project name="optee_test.git" path="optee_test" remote="optee" revision="a00eeb15ddfaa5b7d2cb340faeb36748d5aabffb" upstream="master"/>
  <project name="u-boot.git" path="u-boot" remote="linaro-swg" revision="10577db76e77a513180388df1f176a6642143c84" upstream="rpi3_initial_drop"/>
</manifest>
```